From fad6dc4391fc8e142f2907bc1be09fd53dd788bd Mon Sep 17 00:00:00 2001
From: Michael Limiero <mike5713@gmail.com>
Date: Thu, 2 Jan 2025 10:41:28 -0500
Subject: [PATCH] Correct ACCEL_MOUNT_MATRIX for Chuwi Hi10 Max

(cherry picked from commit c76a7b7c8ca1aa50aaa60cabbaac554717c8d576)

Resolves: RHEL-49810
---
 hwdb.d/60-sensor.hwdb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 3379294a08..8d6a670900 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -304,7 +304,7 @@ sensor:modalias:acpi:MXC6655*:dmi:*:svnCHUWIINNOVATIONLIMITED:pnHi10Go:*
 
 # Chuwi Hi10 Max
 sensor:modalias:acpi:MXC6655*:dmi:*:svnCHUWIInnovationAndTechnology*:pnHi10Max:*
- ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 1
+ ACCEL_MOUNT_MATRIX=0, -1, 0; 1, 0, 0; 0, 0, 1
 
 # Chuwi Hi12
 sensor:modalias:acpi:BOSC0200*:dmi:*:svnHampoo:pnP02BD6_HI-122LP:*
